metapool/prep.py

```diff
@@ -802,6 +806,8 @@ def generate_qiita_prep_file(platedf, seqtype):
                     'runid']
 
     # reorder the dataframe's columns according to the order in the list above.
+    # Note: This will implicitly remove all columns that are not in
```

Comment:
Do we really want this (removing columns not present in the list)? IMOO no but I rather check.

Reply:
resolved! We now let unknown columns pass through.